README.md

```diff
@@ -23,6 +28,13 @@ The Azure App Services extension for VS Code lets you quickly browse, create, ma
 
 * View and edit a web app's files
   * To enable this feature, modify your `appService.showRemoteFiles` user setting to true.
+  * To modify user settings, click File > Preferences > Settings.
+  ![Enable Remote File Editing](resources/Remote_File_Editing_Setting.png)
+
+![Enable Remote File Editing](resources/Remote_File_Editing_Setting.png)
```

Comment:
Still not a fan of this:
![screen shot 2017-12-15 at 11 42 36 am](https://user-images.githubusercontent.com/11282622/34057730-180549f0-e18d-11e7-81a9-a18435daa8e3.png)

It's a lot of big screenshots. Can we make them smaller and maybe have less?